charts/opentelemetry-ebpf/values.yaml

```diff
@@ -0,0 +1,275 @@
+# Default values for opentelemetry-ebpf.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+nameOverride: ""
+fullnameOverride: ""
+
+clusterName: ""
+
+images:
+  tag: "v0.10.0"
+  repository: otel
+  pullPolicy: IfNotPresent
+
+imagePullSecrets: []
+
+resources: {}
+
+otelCollector:
```

Comment:
Does this chart install the OpenTelemetry Collector? Is that a hard requirement?  If not, I'd prefer decoupling this chart from the Collector, which could be installed/managed separately.

Reply:
@TylerHelmuth this chart is not installing the collector, but for the eBPF agent work we need to provide a collector endpoint (address + port) so the collected metrics can be pushed to.

Reply:
It can be any endpoint right? It doesn't have to be an OpenTelemetry Collector?

Reply:
Oh I see your point, we can push this to any backend (e.g Coralogix, Splunk) not necessary a collector.
Is this what you mean?

Reply:
Yes.  Lets use a generic term, like `endpoint` or something, to denote that this is the place the data will be pushed.